Assets/MixedRealityToolkit.Services/InputSystem/FocusProvider.cs

```diff
@@ -791,8 +791,8 @@ private void UpdatePointer(PointerData pointer)
                     LayerMask[] prioritizedLayerMasks = (pointer.Pointer.PrioritizedLayerMasksOverride ?? FocusLayerMasks);
 
                     // Perform raycast to determine focused object
-                    hitResult3d.Clear();
```

Comment:
Is this not necessary anymore?

Reply:
Must have removed by mistake. Thanks for catching.